### PR TITLE
Microsoft Edge Add-ons Publish API What's New and Roadmap

### PR DESCRIPTION
**Pull request by sonakshimore**
Minor change: Publish API moved from Roadmap to Released Features DMC.

Preview links:

- [https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/released-features?branch=pr-en-us-1855](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/released-features?branch=pr-en-us-1855)
- [https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/roadmap?branch=pr-en-us-1855](https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/whats-new/roadmap?branch=pr-en-us-1855)

Please let me know your inputs. Thank you!

### CODE REVIEW

**Review by vchapel — APPROVED** [review]
LGTM

File: microsoft-edge/extensions-chromium/whats-new/released-features.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ This article describes features launched on [Partner Center](https://partner.mic
 For more information about upcoming updates and feature releases, see [Microsoft Edge Add-ons roadmap](roadmap.md).
 
 
+<!-- ====================================================================== -->
+## Publish and update extensions to Microsoft Edge Add-ons using APIs
+
+You can integrate APIs directly into your Continuous Integration / Continuous Delivery (CI/CD) pipeline. You can also publish extension package updates to the Microsoft Edge Add-ons website without using Partner Center. The Microsoft Edge Add-ons Publish API is currently in beta.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/whats-new/released-features.md:23]:
```suggestion
You can integrate APIs directly into your Continuous Integration / Continuous Delivery (CI/CD) pipeline. You can also publish extension package updates to the Microsoft Edge Add-ons website without using Partner Center.
```

File: microsoft-edge/extensions-chromium/whats-new/released-features.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ This article describes features launched on [Partner Center](https://partner.mic
...
+
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/whats-new/released-features.md:24]: link to the Using API article

File: microsoft-edge/extensions-chromium/whats-new/released-features.md (hunk 1 above)
Comment (vchapel) [reply microsoft-edge/extensions-chromium/whats-new/released-features.md:23]: Changed

File: microsoft-edge/extensions-chromium/whats-new/released-features.md (hunk 2 above)
Comment (vchapel) [reply microsoft-edge/extensions-chromium/whats-new/released-features.md:24]: added

File: microsoft-edge/extensions-chromium/whats-new/released-features.md [hunk 3]
```diff
@@ -17,6 +17,14 @@ This article describes features launched on [Partner Center](https://partner.mic
 For more information about upcoming updates and feature releases, see [Microsoft Edge Add-ons roadmap](roadmap.md).
 
 
+<!-- ====================================================================== -->
+## Publish and update extensions to Microsoft Edge Add-ons using APIs
```
Comment (vchapel) [inline microsoft-edge/extensions-chromium/whats-new/released-features.md:21]:
```suggestion
## Publish and update extensions using the Microsoft Edge Add-ons API
```

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**